src/common/OneDSLoggerTelemetry/oneDSLoggerWrapper.ts

```diff
@@ -29,6 +30,7 @@ export class oneDSLoggerWrapper {
 	/// Trace info log
 	public traceInfo(eventName:string, eventInfo?: object, measurement?: object) {
         try{
+            if (!isCustomTelemetryEnabled()) return;
```

Comment:
We should not even initialize 1DSLogger instance

Reply:
Initializing would help handle the scenarios where users midway change their decision & in middle of work-session, enable/disable their telemetry setting. This controls custom telemetry logging - so whenever this would be false, telemetry won't be logged serving our use-case.